model/k8s/registry.yaml

```diff
@@ -232,3 +232,69 @@ groups:
               value: 'terminating'
               brief: "Terminating namespace phase as described by [K8s API](https://pkg.go.dev/k8s.io/api@v0.31.3/core/v1#NamespacePhase)"
               stability: experimental
+      - id: k8s.container.status.state
```

Comment:
are there corresponding K8s APIs that you can link to, similar to in `k8s.namespace.phase` above?